content/docs/zero/upgrading.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Upgrading
+description: This page discusses how to upgrade your cluster to use the latest version of Pomerium Zero.
+keywords:
+  [
+    pomerium zero release,
+    zero release,
+    zero releases,
+    pomerium zero releases,
+    upgrade zero,
+    zero version,
+    latest zero version,
+  ]
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade Pomerium Zero
+
+This page shows you how to upgrade your cluster to use the latest version of Pomerium Zero. Refer to the steps for your deployment environment.
+
+:::note
+
+Review the latest [**Core Upgrade guide**](/docs/deploy/core/upgrading) for release details before you upgrade Pomerium Zero.
+
+:::
+
+## How to upgrade Pomerium Zero
+
+<Tabs>
+<TabItem label="Docker" value="docker">
+
+To update Pomerium Zero in Docker:
+
+1. In your `docker-compose.yaml` file, update the Pomerium Zero image tag to specify the [latest tagged release](https://github.com/pomerium/pomerium/tags):
+
+```yaml
+pomerium:
+  image: pomerium/pomerium:vX.Y.Z
+```
+
+1. Run the following command:
+
+```bash
+$ docker compose up -d
+```
+
+Docker should automatically pull the new image of Pomerium Zero before running the container. If for some reason it doesn't pull the image, you can manually run:
+
+```bash
+$ docker pull pomerium/pomerium:vX.Y.Z
+```
+
+</TabItem>
+<TabItem label="Linux" value="linux">
+
+To update Pomerium Zero in Debian-based Linux systems:
```

Comment:
I think we might want to add a warning here for anyone that used the Linux install script during the beta period (due to https://github.com/pomerium/pomerium-zero/issues/2001). I can work on this.

Reply:
@kenjenkins do you intend to add that notice in this PR or in a separate PR? Either way is fine.

Reply:
Here's one option. (Or alternatively, maybe this would be a good item to add to an FAQ or troubleshooting page, and then we could link to that from here?)

    <details>
    <summary>
    Note: if you installed Pomerium using the Linux install script during the Pomerium Zero beta, you will need to re-run the install script the first time you upgrade Pomerium. (Subsequent upgrades will not require this step.)
    </summary>
    
    First find your current cluster token: look for a line beginning with `Environment=POMERIUM_ZERO_TOKEN=` in the file /usr/lib/systemd/system/pomerium.service. Copy this token into the following command and run it:
    
    ```bash
    $ curl https://console.pomerium.app/install.bash | \
      env POMERIUM_ZERO_TOKEN=<your token here> bash -s install
    ```
    
    </details>

Reply:
I like the idea of linking to the Zero FAQ page from here. When it's merged, we can add this.

Reply:
@kenjenkins I added this as an entry to the troubleshooting page in https://github.com/pomerium/documentation/pull/1426. Feel free to review if you'd like, I'm holding off on merging for a bit.